packages/flutter/test/widgets/dismissible_test.dart

```diff
@@ -798,4 +798,53 @@ void main() {
       '   handler has fired.\n',
     );
   });
+
+  testWidgets('How Dismissible should behave during hit testing', (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('Dismissible.behavior should correctly control hit testing', (WidgetTester tester) async {
```
or something more readable.

Reply:
Do you recommend I rename it to behavior instead of hitTestBehavior @dkwingsmt ? Also, what do you mean by test the default argument? I do test for HitTestBehavior.opaque. Thanks 😄 !

Reply:
I didn't notice you were using `hitTestBehavior`. I do recommend renaming it to `behavior`, since it seems to be the convention of most widgets (all widgets but platform views).

We want to make sure that when the `behavior` argument is not provided, `Dismissible` is acting the same way as `opaque`, since someone might change the default argument by mistake without getting caught by your current tests.